lib/visitorMap.js

```diff
@@ -151,6 +151,20 @@ const commit = {
   committer: self
 };
 
+const pull_request_commit_comment = {
+  _type: 'pull_request_commit_comment',
+  user: self
+};
+
+
+const pull_request_commit = {
+  _type: 'pull_request_commit',
+  pull_request_commit_comments: collection(pull_request_commit_comment),
+  repo: self,
```

Comment:
We might have discussed this but is it possible to link to the ```pull_request```? We typically only link one level up the hierarchy, so we would link to ```pull_request``` not ```repo```.